std/uni.d

```diff
@@ -8131,6 +8185,7 @@ private auto toCaser(alias indexFn, uint maxIdx, alias tableFn, alias asciiConve
         {
             @property auto save()
             {
+                import std.range.primitives : save;
```

Comment:
> So yes, this is redundant for now, but the idea is that it won't be in a couple releases, when the top-level import is made selective.

I'd suggest to just add popFront, front and friends to that top-level selective import. It's used everywhere anyway. I'm opposed to adding tons of identical fine-grained imports for commonly used symbols especially since the module is already imported by the top-level scope.